README.md

```diff
@@ -1,72 +1,204 @@
+**Use:**
+
+1. Import a package from the [services][1] directory. See [SERVICES.md][] for a list.
+1. Create and authenticate a client with a `New*Client` func, e.g.
+   `c := compute.NewVirtualMachinesClient(...)`.
+1. Invoke API methods using the client, e.g. `c.CreateOrUpdate(...)`.
+1. Handle responses.
+
+[1]: https://github.com/Azure/azure-sdk-for-go/tree/master/services
+
+For example, to create a new VM (substitute your own values for angle brackets):
+
+```go
+import (
+    "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2016-03-30/compute"
+    "github.com/Azure/go-autorest/autorest"
+    "github.com/Azure/go-autorest/autorest/to"
+)
+
+c := compute.NewVirtualMachinesClient("<subscription_ID>")
+c.Authorizer = autorest.NewBearerAuthorizer("<adal.OAuthTokenProvider>")
+
+vm, err := c.CreateOrUpdate(
+  "<resource_group_name>",
+  "<vm_name>",
+  compute.VirtualMachine{
+    Location: to.StringPtr("westsus2"),
+    VirtualMachineProperties: &compute.VirtualMachineProperties{
+      HardwareProfile: &compute.HardwareProfile{
+        VMSize: compute.StandardDS1V2,
+      },
+      StorageProfile: &compute.StorageProfile{
+        ImageReference: &compute.ImageReference{
+          Publisher: to.StringPtr("Canonical"),
+          Offer:     to.StringPtr("UbuntuServer"),
+          Sku:       to.StringPtr("16.04-LTS"),
+          Version:   to.StringPtr("latest"),
+        },
+        OsDisk: &compute.OSDisk{
+          Name: to.StringPtr("osDisk223344"),
+          Vhd: &compute.VirtualHardDisk{
+            URI: to.StringPtr("https://<storage_account_name>.blob.core.windows.net/azsamplesgocompute/<vm_name>.vhd")
+          },
+          CreateOption: compute.DiskCreateOptionTypesFromImage,
+        },
+      },
+      OsProfile: &compute.OSProfile{
+        ComputerName:  to.StringPtr("<vm_name>"),
+        AdminUsername: to.StringPtr("<username>"),
+        AdminPassword: to.StringPtr("<password>"),
+        LinuxConfiguration: &compute.LinuxConfiguration{
+          SSH: &compute.SSHConfiguration{
+            PublicKeys: &[]compute.SSHPublicKey{
+              compute.SSHPublicKey{
+                Path:    to.StringPtr(fmt.Sprintf("/home/%s/.ssh/authorized_keys", "<username>")),
+                KeyData: to.StringPtr("<ssh_public_key>"),
+              },
+            },
+          },
+        },
+      },
+      NetworkProfile: &compute.NetworkProfile{
+        NetworkInterfaces: &[]compute.NetworkInterfaceReference{
+          compute.NetworkInterfaceReference{
+            ID: "<your_nic_urn>",
+            NetworkInterfaceReferenceProperties: &compute.NetworkInterfaceReferenceProperties{
+              Primary: to.BoolPtr(true),
+            },
+          },
+        },
+      },
+    },
+  },
+  nil,
+)
+```
+
+For a complete VM example see
+[github.com/Azure-Samples/azure-sdk-for-go-samples/compute][2].
+
+For examples of using more services see
+[Azure-Samples/azure-sdk-for-go-samples][3].
```

Comment:
IMHO, having both of these links feels redundant. I'd rephrase to something like:
``` Markdown
For complete examples, including details on VM management, see:
[Azure-Samples/azure-sdk-for-go-samples](https://github.com/Azure-Samples/azure-sdk-for-go-samples)
```

Reply:
done! i bumped the link to the top of the "use" section too.